bin/node/cli/src/command.rs

```diff
@@ -92,7 +92,7 @@ pub fn run() -> Result<()> {
 				runner.sync_run(|config| cmd.run::<Block, ExecutorDispatch>(config))
 			} else {
 				Err("Benchmarking wasn't enabled when building the node. \
-				You can enable it with `--features runtime-benchmarks`."
+				  You can enable it with `--features runtime-benchmarks`."
```

Comment:
Why have you changed this?

Reply:
It should be a modification made by the plugin